permissions/plugin-pipeline-maven.yml

```diff
@@ -2,5 +2,7 @@
 name: "pipeline-maven"
 paths:
 - "org/jenkins-ci/plugins/pipeline-maven"
+- "org/jenkins-ci/plugins/pipeline-maven-parent"
```

Comment:
isn't it pipeline-maven-pom ? https://github.com/jenkinsci/pipeline-maven-plugin/blob/master/pom.xml#L36 @alvarolobato